src/main/java/com/cloudbees/jenkins/plugins/BitBucketTrigger.java

```diff
@@ -82,8 +84,13 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new BitBucketPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
-                        LOGGER.info("SCM changes detected in "+ job.getName()+". Triggering "+name);
+                    ParameterizedJobMixIn pJob = new ParameterizedJobMixIn() {
+                        @Override protected Job asJob() {
+                            return job;
+                        }
+                    };
+                    if (pJob.scheduleBuild(cause)) {
```

Comment:
@oleg-nenashev This is the way to schedule a build given a Job instance (which doesn't have ```schedule*``` methods)